extensions/sql-migration/src/wizard/skuRecommendation/assessmentSummaryCard.ts

```diff
@@ -0,0 +1,427 @@
+					'color': ColorCodes.NotReadyState_Red,
+				};
+				this._notReadyText = value;
+				break;
+			case AssessmentResultType.BLOCKERS:
+				label.value = constants.BLOCKERS;
+				value.CSSStyles = {
+					...styles.ASSESSMENT_SUMMARY_CARD_CSS,
+				};
+				this._blockersText = value;
+				break;
+			case AssessmentResultType.WARNINGS:
+				label.value = constants.WARNINGS;
+				value.CSSStyles = {
+					...styles.ASSESSMENT_SUMMARY_CARD_CSS,
+				};
+				this._warningsText = value;
+				break;
+		}
+
+		container.addItems([label, value]);
+		return container;
+	}
+
+	// Creates the SKU recommendation Part of summary.
+	// TODO - Add the dialog link later which gives the detail of SKU Recommendation.
+	private createRecommendedConfigurationContainer(view: azdata.ModelView) {
+		const container = view.modelBuilder.flexContainer().withProps({
+			CSSStyles: {
+				'display': 'flex',
+				'flex-direction': 'column',
+				'gap': '4px'
+			}
+		}).component();
+
+		const recommendedConfigurationLabel = view.modelBuilder.text().withProps({
+			value: constants.RECOMMENDED_CONFIGURATION.toLocaleUpperCase(),
+			description: "", // TODO - need this value later
+			height: 18,
+			CSSStyles: {
+				...styles.TOOLBAR_CSS,
+				'text-align': 'center',
+				'align': 'center',
+				'margin-left': '20px',
+			},
+		}).component();
+
+		this._recommendedConfigurationText = view.modelBuilder.text().withProps({
+			value: "",
+			height: 18,
+			CSSStyles: {
+				'font-size': '13px',
+				'line-height': '18px',
+				'font-weight': '600',
+				'margin': '0px',
+			},
+		}).component();
+
+		this._vmRecommendedConfigurationText = view.modelBuilder.text().withProps({
+			value: "",
+			height: 28,
+			CSSStyles: {
+				'font-size': '10px',
+				'line-height': '14px',
+				'font-weight': '400',
+				'margin': '0px',
+			},
+		}).component();
+
+		const viewDetailsLink = view.modelBuilder.hyperlink().withProps({
+			label: constants.VIEW_DETAILS,
+			url: '',
+			height: 18,
+			CSSStyles: {
+				'font-size': '13px',
+				'font-weight': '400',
+				'line-height': '18px',
+				'text-decoration': 'none',
+			}
+		}).component();
+
+
+		container.addItem(recommendedConfigurationLabel);
+		container.addItem(this._recommendedConfigurationText);
+
+		switch (this.migrationTargetType) {
+			case MigrationTargetType.SQLVM:
+				container.addItem(this._vmRecommendedConfigurationText);
+				container.addItem(viewDetailsLink);
+				break;
+			default:
+				container.addItem(viewDetailsLink);
+				container.addItem(this._vmRecommendedConfigurationText);
+		}
+
+		return container;
+	}
+
+	// Used to update the summary of assessment card later after its initialization.
+	// TODO - We can remove dummayData input later and use a some other way to pass the values. Once we start implementing the whole page.
+	public async updateContent(dummyData: DummyData) {
+		await this._assessmentResultText.updateProperties({ "value": constants.ASSESSED_DBS(dummyData.assessmentResult) });
```

Comment:
recommend using the .value property directly instead unless you need it to be asynchronously updated.
as this._assessmentResultText.value("new value")